### DISCUSSION TITLE
How do I connect pouchdb with rust?

### DISCUSSION THREAD

**Discussion by namaljayathunga** [tauri-apps/tauri#3653] [Q&A]
Is there any good library to connect the pouchdb with rust?

**Comment by FabianLars** [comment]
as far as i know, no. Because it's a db designed for JS/the browser (and also running in the browser) connecting to it from Rust is a little tricky.
For Tauri specifically you could probably do something with [events](https://tauri.app/v1/guides/features/events/) 🤔

**Comment by hoichi** [comment]
I was thinking about PouchDB, and the way I see it, our best bet is connecting to SQLite using a plugin like [tauri-plugin-sqlite](https://github.com/lzdyes/tauri-plugin-sqlite) (haven’t been updated for a while, but the basics work) and then, IDK, hack a WebSQL adapter? Because it probably won’t work OOTB.
